### DISCUSSION TITLE
Change width size of collums

### DISCUSSION THREAD

**Discussion by Nuckerr** [squidfunk/mkdocs-material#2842] [Q&A] [answered]
Hello, so I was writing a document and I wanted to use tables. One of the columns inside of my tables I have put a short description however it cant have more than a couple words per line. Is there a way to increase the size of a single column (even if it requires scroll bar) so I can fit at least a few more words per line

**Comment by jromal** [comment, marked as answer]
Try this as column name in the column you want, changing name and width in em.

`<div style="width:20em">Column Name</div>`
